README.md

```diff
@@ -107,6 +107,33 @@ cached_key!{
 }
 ```
 
+4. The `cached_result` and `cached_key_result` macros function similarly to `cached`
```

Comment:
Can you move this to the crate documentation in `src/lib.rs` and (if you can, you don't have to, requires `cargo-readme`) run the `readme.sh` script to regen the README.md